Source/WebKit/UIProcess/API/Cocoa/_WKSystemPreferencesInternal.h

```diff
@@ -26,7 +26,7 @@
 #import "_WKSystemPreferences.h"
 #import <wtf/text/ASCIILiteral.h>
 
-constexpr auto LDMEnabledKey = "LDMGlobalEnabled";
+const auto LDMEnabledKey = CFSTR("LDMGlobalEnabled");
```

Comment:
I’m not sure that this is more efficient than the old idiom. Might be more elegant to not have the bridge cast, but I believe `@("")` is implemented entirely at compile time and `CFSTR` has some runtime cost. That’s also why we can’t use `constexpr` any more.

We should be cautious about changing coding style to something more elegant that might in fact be less efficient.

Reply:
Also note that a bridge cast does not involve copying, so this is a part of the pull request that goes beyond the title, which refers to eliminating unnecessary copying.